### ISSUE TITLE
MGRS coords labeling - bug?

### ISSUE THREAD

**Issue by oncoma** [trailbehind/leaflet-grids#5]
Hello,

I would like to ask, if everything is ok with mgrs map coords labeling/or i just missed something? When I compared mgrs leaflet-grid with grid from mappingsupport, coords are different.

..and if there is a way to have more detailed mgrs grid like in example (red lines
![mappingsupport](https://cloud.githubusercontent.com/assets/26813008/24543182/ed79e3c6-15fe-11e7-9354-d33371965e93.png)
![leaflet-mgrs](https://cloud.githubusercontent.com/assets/26813008/24543183/ed7c193e-15fe-11e7-8b08-940c06958396.png)
![leaflet-mgrs-grid-example](https://cloud.githubusercontent.com/assets/26813008/24543187/f14a1e76-15fe-11e7-9534-dc32b0b3ba49.png)

)?

**Comment by JesseCrocker** [comment]
Hello,
To get lines like in the last screenshot you posted you could change the spacing between grid lines, by modifying or replacing the _gridspacing method, see https://github.com/trailbehind/leaflet-grids/blob/master/leaflet-grids.js#L311

**Comment by JesseCrocker** [comment]
As for the label coordinates being wrong, i think you're correct, the labels do appear to be wrong in this location.

**Comment by oncoma** [comment]
This can be fixed?

**Comment by JesseCrocker** [comment]
Yes, it can be fixed. It's not a high priority for anyone on our team right now, but we will probably get to it eventually. If you would like to try fixing it, we would welcome a pull request.

**Comment by oncoma** [comment]
Yes, i just made a little research and changing (mgrs.js / encode fn line 371 ) 
"... seasting.substr(0, accuracy) + snorthing.substr(0, accuracy) .."
with
".. seasting.substr(1, accuracy) + snorthing.substr(2, accuracy); "...

is fixing this problem, i wil try do some more tests to check if this/or other methods work properly
